src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeInterface.scala

```diff
@@ -39,22 +60,25 @@ trait TezosRPCInterface {
 object TezosNodeInterface extends TezosRPCInterface with LazyLogging {
 
   private val conf = ConfigFactory.load
-  private val awaitTime = conf.getInt("dbAwaitTimeInSeconds").seconds
-  private val entityGetTimeout = conf.getInt("GET-ResponseEntityTimeoutInSeconds").seconds
-  private val entityPostTimeout = conf.getInt("POST-ResponseEntityTimeoutInSeconds").seconds
+  private[tezos] val awaitTime = conf.getInt("dbAwaitTimeInSeconds").seconds
```

Comment:
I don't see a tezos "type" per say. What is this supposed to enforce? That we're in the configuration for tezos specifically?

Reply:
It limits visibility scope of this `val` to the containing package, i.e.
` package tech.cryptonomic.conseil.tezos `

This lets me access the value in `TezosNodeOperator` where I need the `entityGetTimeout`